pandapower/plotting/simple_plot.py

```diff
@@ -173,14 +177,84 @@ def simple_plot(net, respect_switches=False, line_width=1.0, bus_size=1.0, ext_g
             use_line_geodata=not use_bus_geodata, zorder=12, color=switch_color)
         collections.append(sc)
 
+    patch_count_unique = {}
+    sgen_types = {}
+
+    for i in net.bus_geodata.index:
+        sgen_count = 0
+        gen_count = 0
+        load_count = 0
+        if plot_sgens and len(net.sgen):
+            sgen_types_counts = net.sgen[net.sgen.bus == i].type.value_counts()
```

Comment:
To me this is a lot of hard coded and rather implicit indication of the patch types drawn for a simple_plot. I am not too sure how users would react to that. I could agree on this, because it is "just" simple_plot, but in the dedicated plotting functions, we need room for adaptions by the user.